src/shared/modules/udc/udcDuck.ts

```diff
@@ -378,6 +379,30 @@ export const trackCommandUsageEpic: Epic<Action, GlobalState> = action$ =>
 
     const type = cmdHelper.interpret(action.cmd.slice(1))?.name
 
+    if (type === 'play') {
+      const guideName = action.cmd.substr(':play'.length).trim()
+
+      const content = isPlayChapter(guideName) ? 'built-in' : 'non-built-in'
+
+      return metricsEvent({
```

Comment:
Would be nice to remove the extra returns here, and just use an extra `extraDataProps = {}` that you spread into the last return...